src/App.js

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from 'react';
```

Comment:
Add import for React

```suggestion
import React, { useEffect, useState } from 'react';
```